backend/fastrtc/pause_detection/humaware.py

```diff
@@ -0,0 +1,160 @@
+import json
+import logging
+import numpy as np
+import torch
```

Comment:
Hi @CuriousMonkey7 ! Let's not add this to the library yet. I don't want to have to have to add torch as a dependency. 

Instead add an entry for the humaware model here: https://github.com/freddyaboulton/fastrtc/blob/main/docs/vad_gallery.md

With instructions on how to install it. But it will live outside of fastrtc package for now.

Reply:
Okay I'll Update the PR

Reply:
Sweet!